impeller/entity/contents/filters/filter_contents.cc

```diff
@@ -37,10 +37,11 @@ std::shared_ptr<FilterContents> FilterContents::MakeGaussianBlur(
     const FilterInput::Ref& input,
     Sigma sigma_x,
     Sigma sigma_y,
-    BlurStyle blur_style,
-    Entity::TileMode tile_mode) {
+    FilterContents::BlurStyle blur_style,
+    Entity::TileMode tile_mode,
+    const std::shared_ptr<Geometry>& geometry) {
```

Comment:
optional, but you could re-arrange the params so that blur_style (mask_blur_style?) and mask_geometry happen last and default them to normal/nullptr - here and in the ctor.

Reply:
done